### DISCUSSION TITLE
Cant connect FTP to cyberduck after puttling login info, i am putting in right info and password

### DISCUSSION THREAD

**Discussion by thebestskinaesthetics** [iterate-ch/cyberduck#14859] [Q&A]
This ticket system is to report bugs and feature requests. For support, visit the [help page](https://cyberduck.io/help/) first.

 * If you have trouble connecting to a server or your login credentials are not valid, try to resolve the issue with the assistance of your hosting service provider. Any such ticket will be closed with a resolution of _thirdparty_.

**Describe the bug**

**To Reproduce**

**Expected behavior**

**Screenshots**

**Desktop (please complete the following information):**
 - OS: [e.g. macOS]

**Log Files**
Please include the [log file](https://docs.cyberduck.io/cyberduck/support/#logging-output). You can reach the logging output in _Preferences → Connection_.

**Additional context**
<img width="439" alt="Screenshot 2023-06-09 at 11 16 03 AM" src="https://github.com/iterate-ch/cyberduck/assets/135924319/e5d6d3a0-91a6-4cb3-9e14-b034dd0118d6">

**Comment by tatze96** [comment]
Please add the following information:
* A screenshot of the error message.
* The used operating system and version of Cyberduck.
* The Log file: [Enable debug logging](https://docs.cyberduck.io/cyberduck/support/#debug-logging-level), try to connect to the server, and send us the log file afterward. 

If you don't like to see sensitive data to be shared publicly, please send the log file with the ticket reference GH-14859 to support@cyberduck.io, otherwise attach it here.

**Sidenote**: If you want to share the log file and screenshot here, make sure to attach them through the GitHub web interface, as they wouldn't be visible to us otherwise.
